mcoding_bot/cogs/starboard.py

```diff
@@ -46,6 +46,11 @@ def embed_message(
         f"[Go to Message](https://discord.com/channels/"
         f"{bot.config.mcoding_server}/{msg.channel_id}/{msg.id})"
     )
+    if _obj_or_none(msg.attachments):
+        embed.set_image(msg.attachments[0].url)
+        embed.description = embed.description or f"*{msg.attachments[0].filename}*"
```

Comment:
I would just remove this line. It doesn't actually work ("*file only*" is set when there's no content, so embed.description will never be None), and imo it's better for it to say *file only* than the filename (which is usually a bunch of random chars.

If you really want it it's fine with me. Just make sure that it works (right now it doesn't, see explanation above): https://circuit.is-from.space/Screen_Shot_2022-02-01_at_10.57.35_AM.png

Reply:
In my efforts to fix git messing stuff up and reverting my commit, I forgot to remove the `file only`. Without that it works as intended. If either you two could do it or I'll do it in ~3 hours